ament_pep257/ament_pep257/main.py

```diff
@@ -25,9 +25,22 @@
 from xml.sax.saxutils import quoteattr
 
 from pydocstyle import check
-from pydocstyle import ConfigurationParser
-from pydocstyle import Error
-from pydocstyle import log
+
+try:  # as of version 1.1.0
+    from pydocstyle.config import ConfigurationParser
+except ImportError:  # try version 1.0.0
+    from pydocstyle import ConfigurationParser
+
+try:  # as of version 1.1.0
+    from pydocstyle.violations import Error
+except ImportError:  # try version 1.0.0
+    from pydocstyle import Error
+
+try:  # as of version 1.1.0
+    from pydocstyle.utils import log
+except ImportError:  # try version 1.0.0
+    from pydocstyle import log
+
```

Comment:
You can put all new imports in a single try block and all old imports in the single except block.